understanding/index.html

```diff
@@ -466,23 +466,23 @@ <h3><a href="compatible"><span class="secno">4.1 </span>Compatible</a></h3>
             
             <section id="other">
                
-               <h2>Other Understanding documents</h2>
+               <h2>その他の解説書の文書</h2>
                
                <ul class="toc-wcag-docs">
                   
-                  <li><a href="intro">Introduction to Understanding WCAG </a></li>
+                  <li><a href="intro">WCAG 解説書のイントロダクション</a></li>
                   
-                  <li><a href="understanding-techniques">Understanding Techniques for WCAG Success Criteria</a></li>
+                  <li><a href="understanding-techniques">WCAG 達成基準のテクニックを理解する</a></li>
                   
-                  <li><a href="understanding-act-rules">Understanding Test Rules for WCAG Success Criteria</a></li>
+                  <li><a href="understanding-act-rules">WCAG 達成基準のテストルールを知解する</a></li>
                   
-                  <li><a href="conformance">Understanding Conformance</a></li>
+                  <li><a href="conformance">適合を理解する</a></li>
                   
-                  <li><a href="refer-to-wcag">How to Refer to WCAG  from Other Documents</a></li>
+                  <li><a href="refer-to-wcag">他の文書からの WCAG の参照方法</a></li>
                   
-                  <li><a href="documenting-accessibility-support">Documenting Accessibility Support for Uses of a Web Technology</a></li>
+                  <li><a href="documenting-accessibility-support">ウェブコンテンツ技術の使用法のアクセシビリティサポートを文書化する</a></li>
```

Comment:
「アクセシビリティ サポート」とスペースが入るのが正しいはず

```suggestion
                  <li><a href="documenting-accessibility-support">ウェブコンテンツ技術の使用法のアクセシビリティ サポートを文書化する</a></li>
```